dbengine/utils_test.go

```diff
@@ -1,28 +1,12 @@
 package dbengine
 
 import (
-	composeapi "github.com/compose/gocomposeapi"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
-	"github.com/alphagov/paas-compose-broker/compose/fakes"
 )
 
 var _ = Describe("Broker utility functions", func() {
-
 	Describe("findDeployment", func() {
```

Comment:
This Describe block was accidentally left here.

Reply:
This change was only done to remove unused imports and setup of the fake, when the fake changed. We think that restructuring this test is outside the scope of this PR.